src/c4v/scraper/crawler/crawlers/primicia_crawler.py

```diff
@@ -0,0 +1,21 @@
+"""
+    Class for scraping primicia data
+"""
+from c4v.scraper.crawler.crawlers.base_crawler import BaseCrawler
+
+
+class PrimiciaCrawler(BaseCrawler):
+    """
+        Class to crawl Primicia Urls
+    """
+
+    start_sitemap_url = "https://primicia.com.ve/sitemap_index.xml"
```

Comment:
Would suggest making `https://primicia.com.ve` a static constant, as we repeat it a few times

Reply:
So, instead of a `start_sitemap_url`, we would have something like 
```
sitemap_base_url = CONSTANT_URL
sitemap_path = "/sitemap_index.xml"
```
?

Or maybe just 
```
start_sitemap_url = CONSTANT_URL + "/sitemap_index.xml"
```

The reason why I did it like this is because I didn't know if all sitemaps will have the same base url as the site they are mapping, but maybe that's part of the definition of a sitemap?